### PR TITLE
Add oak-testnet.json

### PR DESCRIPTION
**Pull request by imstar15**
https://github.com/paritytech/polkadot/blob/master/node/service/res/polkadot.json

类似的polkadot.json文件

### CODE REVIEW

File: node/cli/src/chain_spec.rs [hunk 1]
```diff
@@ -140,13 +145,13 @@ fn testnet_config_genesis() -> GenesisConfig {
 }
 
 /// oak testnet config.
```
Comment (chrisli30) [inline node/cli/src/chain_spec.rs:147]: 把这句注释换了
/// Generate a json file as configuration template, which is called staging in Polkadot

File: node/cli/src/chain_spec.rs [hunk 2]
```diff
@@ -140,13 +145,13 @@ fn testnet_config_genesis() -> GenesisConfig {
...
-pub fn testnet_config() -> ChainSpec {
+pub fn oak_testnet_staing_testnet_config() -> ChainSpec {
```
Comment (chrisli30) [inline node/cli/src/chain_spec.rs:148]: oak_testnet_staging_config

File: node/cli/src/chain_spec.rs [hunk 3]
```diff
@@ -140,13 +145,13 @@ fn testnet_config_genesis() -> GenesisConfig {
...
 	let boot_nodes = vec!["/dns/testnet.oak.tech/tcp/30333/p2p/12D3KooWBpDWtYunHni9Bz3f4fZt9PZ6Cw8uNKLdiBF5NJbQPPVt".parse().unwrap()];
 	ChainSpec::from_genesis(
-		"OAK Testnet",
-		"testnet",
+		"OAK Staging Testnet",
+		"oak_staging_testnet",
 		ChainType::Live,
-		testnet_config_genesis,
+		oak_testnet_staing_testnet_config_genesis,
```
Comment (chrisli30) [inline node/cli/src/chain_spec.rs:154]: oak_testnet_staging_config_genesis

File: node/cli/src/chain_spec.rs [hunk 4]
```diff
@@ -140,13 +145,13 @@ fn testnet_config_genesis() -> GenesisConfig {
 }
 
 /// oak testnet config.
-pub fn testnet_config() -> ChainSpec {
+pub fn oak_testnet_staing_testnet_config() -> ChainSpec {
 	let boot_nodes = vec!["/dns/testnet.oak.tech/tcp/30333/p2p/12D3KooWBpDWtYunHni9Bz3f4fZt9PZ6Cw8uNKLdiBF5NJbQPPVt".parse().unwrap()];
 	ChainSpec::from_genesis(
-		"OAK Testnet",
-		"testnet",
+		"OAK Staging Testnet",
```
Comment (chrisli30) [inline node/cli/src/chain_spec.rs:151]: OAK Testnet Staging

File: node/cli/src/chain_spec.rs [hunk 5]
```diff
@@ -140,13 +145,13 @@ fn testnet_config_genesis() -> GenesisConfig {
...
+		"oak_staging_testnet",
```
Comment (chrisli30) [inline node/cli/src/chain_spec.rs:152]: oak_testnet_staging

File: node/cli/src/command.rs [hunk 6]
```diff
@@ -55,7 +55,8 @@ impl SubstrateCli for Cli {
 				"dev" => Box::new(chain_spec::development_config()),
 				"local" => Box::new(chain_spec::local_testnet_config()),
 				"fir" | "flaming-fir" => Box::new(chain_spec::flaming_fir_config()?),
-				"testnet" => Box::new(chain_spec::testnet_config()),
+				"oak-testnet" => Box::new(chain_spec::oak_testnet_config()?),
+				"oak-testnet-staging" => Box::new(chain_spec::oak_testnet_staing_testnet_config()),
```
Comment (chrisli30) [inline node/cli/src/command.rs:59]: oak_testnet_staging_config()

File: node/cli/src/chain_spec.rs [hunk 7]
```diff
@@ -78,7 +83,7 @@ fn session_keys(
 	SessionKeys { grandpa, babe, im_online, authority_discovery }
 }
 
-fn testnet_config_genesis() -> GenesisConfig {
+fn oak_testnet_staing_testnet_config_genesis() -> GenesisConfig {
```
Comment (chrisli30) [inline node/cli/src/chain_spec.rs:86]: oak_testnet_staging_genesis()

**Review by chrisli30 — CHANGES_REQUESTED** [review]

File: node/cli/src/chain_spec.rs (hunk 1 above)
Comment (imstar15) [reply node/cli/src/chain_spec.rs:147]: Fixed

File: node/cli/src/chain_spec.rs (hunk 2 above)
Comment (imstar15) [reply node/cli/src/chain_spec.rs]: Fixed

File: node/cli/src/chain_spec.rs (hunk 3 above)
Comment (imstar15) [reply node/cli/src/chain_spec.rs]: Fixed

File: node/cli/src/chain_spec.rs (hunk 4 above)
Comment (imstar15) [reply node/cli/src/chain_spec.rs]: Fixed

File: node/cli/src/chain_spec.rs (hunk 5 above)
Comment (imstar15) [reply node/cli/src/chain_spec.rs]: Fixed

File: node/cli/src/command.rs (hunk 6 above)
Comment (imstar15) [reply node/cli/src/command.rs]: Fixed

File: node/cli/src/chain_spec.rs (hunk 7 above)
Comment (imstar15) [reply node/cli/src/chain_spec.rs]: Fixed

**Review by chrisli30 — APPROVED** [review]
Looks good 👍

**Pull request merged**